nano/node/node.hpp

```diff
@@ -168,7 +168,8 @@ class node final : public std::enable_shared_from_this<node>
 	nano::node_observers observers;
 	nano::port_mapping port_mapping;
 	nano::block_processor block_processor;
-	nano::confirmation_height_processor confirmation_height_processor;
+	std::shared_ptr<nano::confirming_set> confirming_set_impl;
```

Comment:
Can this be unique_ptr? Understanding shared_ptr lifetime is generally harder with the assumption that it can outlive the parent node, is this the case here?

Reply:
This was unintentional.  Fixed.